<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_record_widget</class>
 <widget class="QWidget" name="add_record_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>29</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>  名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startBtn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>开始录制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="endBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>结束录制</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
